client/client_test.go

```diff
@@ -1318,6 +1319,44 @@ func TestClient_UpdateNodeFromFingerprintKeepsConfig(t *testing.T) {
 	assert.Equal(t, 100, client.config.Node.NodeResources.Networks[0].MBits)
 }
 
+// Support multiple IP addresses (ipv4 vs. 6, e.g.) on the configured network interface
+func Test_UpdateNodeFromFingerprintMultiIP(t *testing.T) {
+	t.Parallel()
+
+	var dev string
+	switch runtime.GOOS {
+	case "linux":
+		dev = "lo"
+	case "darwin":
+		dev = "lo0"
```

Comment:
Curious what happens in our testing if we always pass `lo` in test?  Do we verify somewhere that the interface exists?

Reply:
@notnoop I'm not sure where we verify it, but the interface must be available on the machine and the addresses have to be valid too. The fingerprinter will raise errors otherwise

Reply:
oh well then :( .